base/iostream.jl

```diff
@@ -292,15 +292,12 @@ function open(fname::String; lock = true,
     if !lock
         s._dolock = false
     end
-    if ccall(:ios_file, Ptr{Cvoid},
-             (Ptr{UInt8}, Cstring, Cint, Cint, Cint, Cint),
-             s.ios, fname, flags.read, flags.write, flags.create, flags.truncate) == C_NULL
-        systemerror("opening file \"$fname\"")
```

Comment:
Looks like this change is the actual issue here. Just replace this with `$(repr(fname))` as it was before?

Reply:
https://github.com/JuliaLang/julia/pull/53813